### ISSUE TITLE
Generate deployment manifest ERROR

### ISSUE THREAD

**Issue by dannycrief** [sassoftware/viya4-deployment#452]
Trying to install Viya: 2023.04 stable version
Running the following code:
```shell
sudo docker run --rm \
  --group-add root \
  --user $(id -u):$(id -g) \
  --volume $HOME/workspace/assets/:/data \
  --volume $HOME/workspace/ansiblevars.yaml:/config/config \
  --volume $HOME/workspace/terraform.tfstate:/config/tfstate \
  --volume $HOME/.ssh/id_rsa:/config/jump_svr_private_key \
  --volume $HOME/.kube/config:/config/kubeconfig \
  viya4-deployment --tags "viya,install"
```
And receiving the following error:
```shell
TASK [vdm : kustomize - Generate deployment manifest] **************************
fatal: [localhost]: FAILED! => changed=true
  cmd:
  - kustomize
  - build
  - /data/skozurak19viya4-aks/sas-viya
  - --load_restrictor=none
  - -o
  - /data/skozurak19viya4-aks/sas-viya/site.yaml
  delta: '0:00:09.289961'
  end: '2023-06-04 15:55:20.868725'
  msg: non-zero return code
  rc: 1
  start: '2023-06-04 15:55:11.578764'
  stderr: 'Error: accumulating resources: accumulateFile "accumulating resources from ''sas-bases/overlays/crunchydata/postgres-operator'': read /data/skozurak19viya4-aks/sas-viya/sas-bases/overlays/crunchydata/postgres-operator: is a directory", accumulateDirector: "recursed accumulation of path ''/data/skozurak19viya4-aks/sas-viya/sas-bases/overlays/crunchydata/postgres-operator'': accumulating resources: accumulateFile \"accumulating resources from ''install/singlenamespace'': read /data/skozurak19viya4-aks/sas-viya/sas-bases/overlays/crunchydata/postgres-operator/install/singlenamespace: is a directory\", accumulateDirector: \"couldn''t make target for path ''/data/skozurak19viya4-aks/sas-viya/sas-bases/overlays/crunchydata/postgres-operator/install/singlenamespace'': json: unknown field \\\"labels\\\"\""'
  stderr_lines: <omitted>
  stdout: ''
  stdout_lines: <omitted>
```

Code I used:

- Newest version of https://github.com/sassoftware/viya4-iac-azure
```shell
git clone https://github.com/sassoftware/viya4-iac-azure $HOME/viya4-iac-azure
sudo docker build -t viya4-iac-azure $HOME/viya4-iac-azure
sudo docker run --rm --group-add root \
  --user "$(id -u):$(id -g)" \
  --volume=$HOME/.ssh:/.ssh \
  --volume=$HOME/workspace:/workspace \
  viya4-iac-azure \
  plan -var-file=/workspace/terraform.tfvars \
       -state=/workspace/terraform.tfstate \
       -out=/workspace/terraform.tfplan
sudo docker run --rm --group-add root \
  --user "$(id -u):$(id -g)" \
  --volume=$HOME/.ssh:/.ssh \
  --volume=$HOME/workspace:/workspace \
  viya4-iac-azure \
  apply -auto-approve \
        -state=/workspace/terraform.tfstate \
        /workspace/terraform.tfplan
sudo docker run --rm --group-add root \
  --user "$(id -u):$(id -g)" \
  --volume=$HOME/workspace:/workspace \
  viya4-iac-azure \
  output -state=/workspace/terraform.tfstate
```
- Project https://github.com/sassoftware/viya4-deployment version 5.6.0 (also I tried to use 6.6.0)
```shell
git clone https://github.com/sassoftware/viya4-deployment --branch 5.6.0 $HOME/viya4-deployment # or 6.6.0
sudo docker build -t viya4-deployment \
  --build-arg kustomize_version=3.7.0 \
  --build-arg kubectl_version=1.23.8 \
  $HOME/viya4-deployment
sudo docker run --rm \
  --group-add root \
  --user $(id -u):$(id -g) \
  --volume $HOME/workspace/assets/:/data \
  --volume $HOME/workspace/ansiblevars.yaml:/config/config \
  --volume $HOME/workspace/terraform.tfstate:/config/tfstate \
  --volume $HOME/.ssh/id_rsa:/config/jump_svr_private_key \
  --volume $HOME/.kube/config:/config/kubeconfig \
  viya4-deployment --tags "baseline,install"
nfs_public_ip=$(cat $HOME/workspace/terraform.tfstate | jq -r '.outputs.nfs_public_ip.value' | tr -d '"')
ssh nfsuser@$nfs_public_ip mkdir -p /export/sas-viya/bin
ssh nfsuser@$nfs_public_ip mkdir -p /export/sas-viya/data
ssh nfsuser@$nfs_public_ip mkdir -p /export/sas-viya/homes
ssh nfsuser@$nfs_public_ip mkdir -p /export/sas-viya/astores
sudo docker run --rm \
  --group-add root \
  --user $(id -u):$(id -g) \
  --volume $HOME/workspace/assets/:/data \
  --volume $HOME/workspace/ansiblevars.yaml:/config/config \
  --volume $HOME/workspace/terraform.tfstate:/config/tfstate \
  --volume $HOME/.ssh/id_rsa:/config/jump_svr_private_key \
  --volume $HOME/.kube/config:/config/kubeconfig \
  viya4-deployment --tags "viya,install" # Here I got error
```

**Comment by dannycrief** [comment]
Here's my ansiblevars.yaml file:
```yaml
## Cluster
NAMESPACE: sas-viya

## MISC
DEPLOY: true # Set to false to stop at generating the manifest
LOADBALANCER_SOURCE_RANGES: ["ip-address"]

## Storage
V4_CFG_MANAGE_STORAGE: true
# JUMP_SVR_RWX_FILESTORE_PATH: /export

## SAS API Access
V4_CFG_SAS_API_KEY: 'myKey'
V4_CFG_SAS_API_SECRET: 'mySecret'
V4_CFG_ORDER_NUMBER: myOrderNumber

V4_CFG_DEPLOYMENT_ASSETS: ViyaArchive.tgz
V4_CFG_CADENCE_VERSION: 2023.04
V4_CFG_CADENCE_NAME: stable

## CR Access
#V4_CFG_CR_USER: viyacregistry
#V4_CFG_CR_PASSWORD: <pass>
#V4_CFG_CR_URL: viyacregistry.azurecr.io

## Ingress
V4_CFG_INGRESS_TYPE: ingress
V4_CFG_INGRESS_FQDN: my-hostname.eastus.cloudapp.azure.com
V4_CFG_TLS_MODE: "full-stack" # [full-stack|front-door|ingress-only|disabled]

## Postgres
V4_CFG_POSTGRES_SERVERS:
  default:
    internal: true   # false

## CAS
# V4_CFG_CAS_WORKER_COUNT: 2

## LDAP
V4_CFG_EMBEDDED_LDAP_ENABLE: true

## Consul UI
V4_CFG_CONSUL_ENABLE_LOADBALANCER: false

## SAS/CONNECT
V4_CFG_CONNECT_ENABLE_LOADBALANCER: false

## Monitoring and Logging
## uncomment and update the below values when deploying the viya4-monitoring-kubernetes stack
#V4M_BASE_DOMAIN: my-hostname.eastus.cloudapp.azure.com
#V4M_STORAGECLASS: sas
#V4M_GRAFANA_PASSWORD: Password123
#V4M_KIBANA_PASSWORD: Password123
#V4M_LOGCOLLECTOR_PASSWORD: Password123

#Opendistro
V4_CFG_ELASTICSEARCH_ENABLE: true
```

**Comment by dhoucgitter** [comment]
Hi @dannycrief, could you try viya4-deployment 6.6.0 again but allow it to build with the default of kustomize 4.5.7? kustomize 3.7.0 is not supported with 2023.04.

See the information below regarding minimum kustomize version for SAS Viya platform 2023.02 or later in the [viya4-deployment:6.2.0 release notes.](https://github.com/sassoftware/viya4-deployment/releases/tag/6.2.0)

Kustomize version:
With SAS Viya platform release cadence version 2023.02, kustomize 4.5.7 is the new [minimum requirement](https://go.documentation.sas.com/doc/en/itopscdc/v_037/itopssr/n1ika6zxghgsoqn1mq4bck9dx695.htm#n0u8dut20wmtp3n1jukmbg0dmim5) for generating Kubernetes manifest files. The default kustomize binary version that is included in the Dockerfile will now be 4.5.7. We do still provide end users the ability to build their viya4-deployment images with a specific kustomize version if the deployment cadence requires it (<=2023.01).
